billchop-fe/src/backend/clients/BillClient.ts

```diff
@@ -0,0 +1,45 @@
+import Axios, { AxiosResponse } from "axios";
+import url from "url-join";
+import BaseClient from "./AbstractClient";
+import Bill from "../models/Bill";
+import { CreateNewBill } from "../models/CreateNewBill";
+
+const groupParam = "groupId";
```

Comment:
Dunno if typescript has same convention as other languages, but shouldn't this be in all caps?

Reply:
Yeah you are right, will change